JSDemos/Demos/Scheduler/CellTemplates/Angular/app/app.component.html

```diff
@@ -19,27 +19,41 @@
   <div
     *dxTemplate="let dataCell of 'dataCellTemplate'"
     [ngClass]="{
-      'disable-date': isDisableDate(dataCell.startDate),
+      'disable-date':
+        (isHoliday | apply : dataCell.startDate : this) ||
+        (isWeekend | apply : dataCell.startDate),
       dinner:
-        !isDisableDate(dataCell.startDate) && isDinner(dataCell.startDate),
-      'dx-scheduler-date-table-cell-text': isMonthView()
+        !(
+          (isHoliday | apply : dataCell.startDate : this) ||
+          (isWeekend | apply : dataCell.startDate)
+        ) && (isDinner | apply : dataCell.startDate : this),
+      'dx-scheduler-date-table-cell-text': currentView === 'month'
     }"
   >
-    {{ isMonthView() ? dataCell.startDate.getDate() : "" }}
+    {{ currentView === "month" ? dataCell.startDate.getDate() : "" }}
```

Comment:
Let's use ```date``` built-in pipe here
[Link to the docs](https://angular.io/api/common/DatePipe)

```suggestion
    {{ currentView === "month" ? (dataCell.startDate | date) : "" }}
```